files/build_templates/sonic_debian_extension.j2

```diff
@@ -109,6 +109,18 @@ sudo rm -rf $FILESYSTEM_ROOT/$REDIS_DUMP_LOAD_PY2_WHEEL_NAME
 # Install Python module for ipaddress
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install ipaddress
 
+# Install Python module for psutil
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install psutil
+
+# Install SwSS SDK Python 3 package
+# Note: the scripts will be overwritten by corresponding Python 2 package
+if [ -e {{swsssdk_py3_wheel_path}} ]; then
+    SWSSSDK_PY3_WHEEL_NAME=$(basename {{swsssdk_py3_wheel_path}})
+    sudo cp {{swsssdk_py3_wheel_path}} $FILESYSTEM_ROOT/$SWSSSDK_PY3_WHEEL_NAME
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install $SWSSSDK_PY3_WHEEL_NAME
+    sudo rm -rf $FILESYSTEM_ROOT/$SWSSSDK_PY3_WHEEL_NAME
+fi
```

Comment:
Why did you add this?

Reply:
That's a mistake. I think this happened when I fixed the conflicts during the cherry-pick. Let me fix it.